runtime/rsyslog.h

```diff
@@ -510,6 +510,9 @@ operation not carried out */
 	/* RainerScript error messages (range 1000.. 1999) */
 	RS_RET_SYSVAR_NOT_FOUND = 1001, /**< system variable could not be found (maybe misspelled) */
 	RS_RET_FIELD_NOT_FOUND = 1002, /**< field() function did not find requested field */
+	// <kortemik>
```

Comment:
I would appreciate if you could remove these author lines. git log shows that and so it only clutters the source.